{{ cookiecutter.project_slug }}/django_module/urls.py

```diff
@@ -9,8 +9,8 @@
     path('admin/', admin.site.urls),
```

Comment:
Noticed yesterday we don't have a URL configured for the robots.txt view! Can you add one?

Reply:
Ah good eye, that's now been added to both url files!